web/src/gamesShared/components/fbg/GameDarkSublayout.tsx

```diff
@@ -106,14 +112,24 @@ export class GameDarkSublayoutInternal extends React.Component<IGameDarkSublayou
           style={{
             position: 'relative',
             width: '100%',
-            maxWidth: this.props.allowWiderScreen ? '1000px' : '500px',
-            color: 'white',
-            top: 'calc(50vh - 49px)',
-            left: '50%',
-            transform: 'translate(-50%, -50%)',
+            height: 'calc(100vh - 49px)',
+            textAlign: 'center',
+            lineHeight: 'calc(100vh - 49px)',
           }}
         >
-          {this.props.children}
+          <div
+            style={{
+              position: 'relative',
```

Comment:
It can be done later, but it would be good to move this style to a css file for increasing readability.